components/pricing_modal/content.tsx

```diff
@@ -258,7 +260,7 @@ function Content(props: ContentProps) {
                                 }
                             },
                             text: formatMessage({id: 'pricing_modal.btn.downgrade', defaultMessage: 'Downgrade'}),
-                            disabled: isStarter || isEnterprise || !isAdmin,
+                            disabled: isStarter || isEnterprise || !isAdmin || !currentSubscriptionIsMonthly,
```

Comment:
Can this be added to the e2e test cases?

Reply:
Yep, added. Thanks